api/swagger.yaml

```diff
@@ -4901,6 +4902,7 @@ paths:
             - `dangling=<boolean>` When set to `true` (or `1`), prune only
                unused *and* untagged images. When set to `false`
                (or `0`), all unused images are pruned.
+            - `until=<string>` Prune images created until this timestamp.
```

Comment:
Thanks. Done. Also updated the part for `/networks/prune` (missed that one before).